projects/valtimo/plugin/src/lib/plugins/open-zaak/open-zaak-plugin.specification.ts

```diff
@@ -28,24 +28,39 @@ const openZaakPluginSpecification: PluginSpecification = {
       description:
         'OpenZaak is een productiewaardig API platform die de API-standaard voor zaakgericht werken implementeert (de ZGW-API’s).',
       configurationTitle: 'Configuratienaam',
+      configurationTitleTooltip:
+        'Onder deze naam zal de plugin te herkennen zijn in de rest van de applicatie',
       clientId: 'Client ID',
+      clientIdTooltip:
+        'Vul hier het clientId in, dat geconfigureerd staat onder OpenZaak beheer onder API authorisaties > Applicaties, dat de juiste authorisaties heeft voor de benodigde functionaliteit',
```

Comment:
Vul hier het clientId in dat geconfigureerd staat onder OpenZaak-beheer (zie API authorisaties > Applicaties). Dit client id moet de juiste authorisaties hebben voor de benodigde functionaliteit.